gcloud-java-core/src/main/java/com/google/gcloud/ServiceOptions.java

```diff
@@ -28,14 +28,7 @@
 import com.google.common.collect.Iterables;
 import com.google.gcloud.spi.ServiceRpcFactory;
 
-import java.io.BufferedReader;
-import java.io.File;
-import java.io.FileReader;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.io.ObjectStreamException;
-import java.io.Serializable;
+import java.io.*;
```

Comment:
Didn't notice it and was done by the IDEA which for some reason reset some of the configuration.